.changeset/calm-icons-jump.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-api': patch
```

Comment:
Made this 2x patch instead of a `minor` bump in core-api, as it would only break usage that was already broken.